matview/dbutils.py

```diff
@@ -0,0 +1,125 @@
+from django.db import migrations
+
+from .models import MatView
+
+
+def _make_matview_migration(src, filters, name):
+    """Generate the SQL strings forward and backwards to create and drop materialized views.
+
+    This create a second view which simply materializes a count of the regular one. Both have
+    a unique index to allow concurrent updates.
+    """
+
+    data_clause = ("'{" + ','.join((
+        '"%s":"%s"' % (k, filters[k])
+        for k in sorted(filters)
```

Comment:
i wanted to make sure the migration it creates is always the same to avoid any confusion in debugging if the same one looked different in test/production/local

Reply:
oh right, I missed that this was stored in the MatView object.